yoti_client.go

```diff
@@ -121,13 +115,13 @@ func (client *Client) getActivityDetails(token string) (activity ActivityDetails
 
 	httpMethod := http.MethodGet
 
-	decryptedToken, err := decryptToken(token, client.Key)
+	decryptedToken, err := cryptoutil.DecryptToken(token, client.Key)
 
 	if err != nil {
 		return activity, errors.New("Unable to decrypt token")
 	}
 
-	endpoint := getProfileEndpoint(decryptedToken, client.GetSdkID())
+	endpoint := web.GetProfileEndpoint(decryptedToken, client.GetSdkID())
```

Comment:
Could we move this into the `profile` package - something like `profile/service.go`, which would handle the request?

The `yoti.Client` can just use these services (same for AML)

Reply:
Moved in db15f25, will make unexported in next commit